mmdet/models/detectors/yolox.py

```diff
@@ -1,12 +1,47 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+import random
+
+import torch
+import torch.distributed as dist
+import torch.nn.functional as F
+from mmcv.runner import get_dist_info
+
 from ..builder import DETECTORS
 from .single_stage import SingleStageDetector
 
 
 @DETECTORS.register_module()
 class YOLOX(SingleStageDetector):
     r"""Implementation of `YOLOX: Exceeding YOLO Series in 2021
-    <https://arxiv.org/abs/2107.08430>`_"""
+    <https://arxiv.org/abs/2107.08430>`_
+
+    Note: Multi-scale training in the model is not recommended.
+    Taking into account the training speed and accuracy, it will
```

Comment:
Considering the trade-off between training speed and accuracy, multi-scale training is temporarily kept.
More elegant implementation will be adopted in the future.